### DISCUSSION TITLE
✅  Clés ECC (ECDSA, Ed25519)

### DISCUSSION THREAD

**Discussion by Diablotin76** [isontheline/pro.webssh.net#506] [Ideas]
Bonjour envisagez vous l’implémentation des clés ECC (ECDSA, Ed25519) 
le RSA c'est bien mais ce n'est plus recommandé 
https://www.ssi.gouv.fr/uploads/2014/01/NT_OpenSSH.pdf  4.1.2 R10
here for the english people ( https://www.ssi.gouv.fr/uploads/2014/01/NT_OpenSSH_en.pdf )
Merci par avance

**Comment by isontheline** [comment]
Bonjour @Diablotin76 👋

Le support est déjà effectif depuis plusieurs mois :
* https://github.com/isontheline/pro.webssh.net/issues/437
* https://github.com/isontheline/pro.webssh.net/issues/5

1. Quelle est l'erreur affichée ?
2. Avez-vous des logs à me fournir ?

Merci infiniment.

Belle journée à vous ☀️

**Comment by Diablotin76** [comment]
impossible d'importer ma cleéSSH depuis icloud ( elle apparait grisé et n'est pas sectionnable) celle-ci à pourtant été générer depuis mon MAC

**Reply by isontheline** [reply]
Grisée ? Sur l'écran d'édition des connexions ?

Est-il possible d'avoir une capture d'écran en masquant les informations sensibles ?

**Comment by Diablotin76** [comment]
Bon j'ai airdrop ma clé direct sur l' iphone direct depuis mon mac et c'est impec, juste l'import depuis icloud qui semble posé souci

**Reply by isontheline** [reply]
Merci beaucoup pour votre retour et votre solution !

Je créer une issue pour vérifier que la sélection puisse s'effectuer correctement : 
https://github.com/isontheline/pro.webssh.net/issues/507
